Support/Sources/Delivery/Support.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Push.swift
+//
+//
+//  Created by Martin Dutra on 10/9/21.
+//
+
+import Foundation
+import UIKit
+
+/// Support is a simple utility to show the Help and Support section
+public class Support {
```

Comment:
This class seems to be mostly boilerplate that just delegates everything to another object. I don't think it necessarily needs to be changed for this PR, but in the future maybe this would be better represented as a protocol.

Reply:
I agree that it looks mostly useless. I think by having this Delivery layer, although a really thin layer in Support, is important to reduce dependency between Planetary and Support, because none of the functions in this Support class depend on classes defined in the Domain layer, they just use classes defined in Foundation or (not quite right) UIKit, so if we change anything of the business logic of Support, it won't impact Planetary.